exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,12 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if ((mark) >= 80) return "A";
+  else if (mark < 80 && mark > 60) return "B";
```

Comment:
Hey Negin,
Hope you are well and having a good week.
I've reviewed your code - well done. I even found a few ways I could improve my own.

I just wanted to mention one way I noticed that you could shorten this function. 

If you look at line 13 you wouldn't need to check that the mark is smaller than 80 as it was already checked at line 12 before passing down to the else statement. Something like: **else if (mark > 60) return "B";** would be enough. You can also do something similar with line 14. 

Hope that makes sense. Let me know if any issues.

Speak soon.

Nigel

Reply:
Hi Nigel,

Thank you for your kind feedbacks. You are very right and this not only helped me have shorten codes, it also helped me understand the conditional logic a bit more clear, I'll change the code. Thank you :)